ee/modules/160-multitenancy-manager/crds/projects.yaml

```diff
@@ -54,9 +61,52 @@ spec:
                 sync:
                   description: Project definition sync with cluster.
                   type: boolean
-              type: object
-          type: object
+    - name: v1alpha2
       served: true
       storage: true
+      deprecated: false
       subresources:
         status: {}
+      additionalPrinterColumns:
+        - jsonPath: .status.sync
+          name: Ready
+          type: boolean
+        - jsonPath: .spec.description
+          name: Description
+          type: string
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                description:
+                  description: Arbitrary description of the purpose of the environment.
+                  type: string
+                projectTypeName:
+                  description: |
+                    The name of the custom resource [ProjectType](cr.html#projecttype) to be used
+                    as a template for creating resources for the environment.
+                  type: string
+                template:
+                  additionalProperties:
+                    x-kubernetes-preserve-unknown-fields: true
+                  description:
+                    Values for [resource templates](cr.html#projecttype-v1alpha1-spec-resourcestemplate) from the environment template (CR [ProjectType](cr.html#projecttype) in helm values format that map to the OpenAPI specification (the [openAPI](cr.html#projecttype-v1alpha1-spec-openapi) parameter of the environment template).
+                  type: object
+                dedicatedNodeName:
```

Comment:
we need some other contract here:
something like
```
dedicatedNodes:
  nodeSelector:
    matchLabels:
      foo: bar
  tolerations:
    - key: dedicated
      operator: Equal
      value: example
```

Reply:
```
            dedicatedNodes:
                type: object
                properties:
                    labelSelector:
                      description: A label selector is a label query over a set
                        of resources. The result of matchLabels and matchExpressions
                        are ANDed. An empty label selector matches all objects.
                        A null label selector matches no objects.
                      properties:
                        matchExpressions:
                          type: array
                          description: matchExpressions is a list of label selector
                            requirements. The requirements are ANDed.
                          items:
                            description: A label selector requirement is a selector
                              that contains values, a key, and an operator that
                              relates the key and values.
                            oneOf:
                              - properties:
                                  operator:
                                    enum: [Exists, DoesNotExist]
                                required: [key, operator]
                                not:
                                  required: [values]
                              - properties:
                                  operator:
                                    enum: [In, NotIn]
                                required: [key, operator, values]
                            properties:
                              key:
                                description: key is the label key that the selector
                                  applies to.
                                type: string
                              operator:
                                description: operator represents a key's relationship
                                  to a set of values. Valid operators are In, NotIn,
                                  Exists and DoesNotExist.
                                type: string
                              values:
                                description: values is an array of string values.
                                  If the operator is In or NotIn, the values array
                                  must be non-empty. If the operator is Exists or
                                  DoesNotExist, the values array must be empty.
                                  This array is replaced during a strategic merge
                                  patch.
                                type: array
                                items:
                                  type: string
                                  pattern: '[a-z0-9]([-a-z0-9]*[a-z0-9])?'
                                  minLength: 1
                                  maxLength: 63
                            type: object
                        matchLabels:
                          additionalProperties:
                            type: string
                          description: matchLabels is a map of {key,value} pairs.
                            A single {key,value} in the matchLabels map is equivalent
                            to an element of matchExpressions, whose key field is
                            "key", the operator is "In", and the values array contains
                            only "value". The requirements are ANDed.
                          type: object
                      type: object
                      x-kubernetes-map-type: atomic
```

Reply:
The contract is saved. 
The "dedicatedNodes" option has been moved to the template